Btms.Model/Auditing/AuditEntry.cs

```diff
@@ -26,7 +27,9 @@ public class AuditEntry
 
     public List<AuditDiffEntry> Diff { get; set; } = new();
     
-    public Dictionary<string, Dictionary<string, string>> Context { get; set; } = new();
+    // TODO - getting a serialisation error when using IAuditContext
```

Comment:
If its a BSON error, then if you put BsonKnownTypes on the interface, i think that fixes it (it auto adds a _t field with the type name so it knows the concrete type to deserialise back into)

Reply:
Attribute 'MongoDB.Bson.Serialization.Attributes.BsonKnownTypesAttribute' is not valid on this declaration type. It is valid on 'Class, Struct' declarations only.

So not valid on either an interface or property :(